src/Kiota.Builder/CodeDOM/CodeConstant.cs

```diff
@@ -4,24 +4,39 @@
 
 namespace Kiota.Builder.CodeDOM;
 
-public class CodeConstant : CodeTerminalWithKind<CodeConstantKind>
+public class CodeConstant : CodeTerminalWithKind<CodeConstantKind>, IDocumentedElement
 {
+    public BlockDeclaration StartBlock { get; set; } = new();
```

Comment:
Adding the `StartBlock` property here makes it seem like `CodeConstant` should now simply inherit from `CodeBlock<BlockDeclaration, BlockEnd>`. Digging into this though, and the base `CodeTerminalWithKind` already being a class, there may be more to do here/resolve. 
Thoughts? (If you agree, I'm open to separate this to a separate issue/PR with the other unintended refactoring that may come along.)

Reply:
@andrueastman the reason I didn't go this way (which I originally explored) is because the code blocks bring a lot more ceremony. You need a writer for the start block, one for the end block, composing blocks into one another is not something kiota is designed for outside of code files, etc.... Constants "feel" like they should remain simple, besides the need to bring usings with them.
Let me know if you still feel like I should change things up. This is always something we can revisit later.

Reply:
True. Constants should remain simple, and the extra ceremony is not worth it here. 
I'm okay with revisiting this later as the usings(via the start block) is something we need for the Constants and we can look at that aspect separately. 
Resolving...